packages/polling-controller/src/PollingController.ts

```diff
@@ -20,10 +34,9 @@ function PollingControllerMixin<TBase extends Constructor>(Base: TBase) {
    *
    */
   abstract class PollingControllerBase extends Base {
-    readonly #networkClientIdTokensMap: Map<NetworkClientId, Set<string>> =
-      new Map();
+    readonly #pollingGroupIds: Map<PollingGroupId, Set<string>> = new Map();
```

Comment:
This sounds like this variable holds an array of ids. What do you think about:

``` suggestion
    readonly #pollTokenGroups: Map<PollingGroupId, Set<string>> = new Map();
```

On the other hand, it seems like when we use this variable we really care about the poll tokens. Also, `intervalIds` and `callbacks` seem to be named after the values that they hold, and so maybe it makes sense for this variable to follow the same pattern:

``` suggestion
    readonly #pollTokens: Map<PollingGroupId, Set<string>> = new Map();
```